bootstrap/ic-shared-minio/create-buckets.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: batch/v1
```

Comment:
Please squash the commits into a single commit since some of the commits are reverting the changes of prior commits (such as removing the minio application manifests and then adding them back)